azure-cli2017.pyproj

```diff
@@ -1220,6 +1232,11 @@
     <Folder Include="command_modules\azure-cli-ams\azure\cli\command_modules\" />
     <Folder Include="command_modules\azure-cli-ams\azure\cli\command_modules\ams\" />
     <Folder Include="command_modules\azure-cli-ams\azure\cli\command_modules\ams\tests\" />
+    <Folder Include="command_modules\azure-cli-managedservices\" />
+    <Folder Include="command_modules\azure-cli-managedservices\azure\" />
+    <Folder Include="command_modules\azure-cli-managedservices\azure\cli\" />
+    <Folder Include="command_modules\azure-cli-managedservices\azure\cli\command_modules\" />
+    <Folder Include="command_modules\azure-cli-managedservices\azure\cli\command_modules\managedservices\" />
```

Comment:
These includes will be broken with the new package structure. Please update.

Reply:
Sure. The cli.pyproj needs to be updated after the structure change but I have moved managedservices to the appropriate location in the solution.